RUFAS/biophysical/feed_storage/feed_manager.py

```diff
@@ -233,6 +233,17 @@ def report_feed_storage_levels(self, simulation_day: int, reporting_suffix: str)
         self.report_stored_farmgrown_feeds(simulation_day, reporting_suffix)
         self.purchased_feed_storage.report_stored_purchased_feeds(simulation_day, reporting_suffix)
 
+    def report_cumulative_purchased_feeds(self, simulation_day: int) -> None:
```

Comment:
This function sets up reporting the `_cumulative_purchased_feeds` data on a daily basis which is needed for emissions reporting flexibility (i.e. the user needs to be able to get cumulative feed amounts at any point, not just on the ration formulation period timeframe).